nncf/common/tensor_statistics/statistic_point.py

```diff
@@ -52,7 +52,7 @@ def add_statistic_point(self, statistic_point: StatisticPoint) -> None:
 
         :param statistic_point: Statistic point to add.
         """
-        target_node_name = statistic_point.target_point.target_node_name
+        target_node_name = statistic_point.target_point.target_node_name  # type: ignore
```

Comment:
```suggestion
        target_node_name: str = statistic_point.target_point.target_node_name
```

Reply:
I committed the change you suggested here, but am receiving the following mypy error again `nncf\common\tensor_statistics\statistic_point.py:55: error: "TargetPoint" has no attribute "target_node_name"  [attr-defined]
Found 1 error in 1 file (checked 6 source files)`

Reply:
@daniil-lyakhov So do I keep it as it is or add `# type: ignore` back?